catalog/app/containers/Bucket/PackageDialog/PackageDialog.tsx

```diff
@@ -341,6 +342,14 @@ export function WorkflowInput({
 export const defaultWorkflowFromConfig = (cfg?: workflows.WorkflowsConfig) =>
   cfg && cfg.workflows.find((item) => item.isDefault)
 
+export const useWorkflowValidator = (workflowsConfig?: workflows.WorkflowsConfig) => (
+  workflow?: workflows.Workflow,
```

Comment:
probably we should memoize returned function by `workflowsConfig?.isWorkflowRequired`